lib/svg_component.dart

```diff
@@ -7,7 +7,8 @@ import './svg.dart';
 class SvgComponent extends PositionComponent {
   Svg svg;
 
-  SvgComponent.fromSvg(Vector2 size, this.svg) {
+  SvgComponent.fromSvg(Vector2 position, Vector2 size, this.svg) {
```

Comment:
actually it should be optional, and you can call the super constructor
```
SvgComponent.fromSvg(this.svg, {Vector2 position, Vector2 size}) : super(position: position, size: size)
```

Reply:
thats more in line with what we do with other PosComp children

Reply:
Ah, yes, much cleaner!

Reply:
So should both be nullable?

Reply:
yes, my bad